emlog.c

```diff
@@ -109,9 +109,13 @@ static struct emlog_info *get_einfo(const struct inode *inode)
     if (inode == NULL)
         return NULL;
 
+    spin_lock(&emlog_info_list_lock); // Peeba
```

Comment:
What does Peeba mean?

Reply:
please ignore it.

Reply:
If I'm going to help maintaining this code, I'd like to understand all comments.